lib/tasks/heroku.rake

```diff
@@ -3,11 +3,11 @@ namespace :heroku do
     HerokuCommands.backup args[:heroku_app]
   end
 
-  task :clone_local, %i[heroku_app backup_id] => :environment do |_t, args|
+  task :clone_local, %i[heroku_app profile backup_id] => :environment do |_t, args|
     # to run this with multiple args, the brackets must be escaped like this:
     # rails "heroku:clone_local[better-way-ars,a1010]"
```

Comment:
Can you update this comment to include some examples of what this command looks like now with the new profile command? This is a good place for reference for future